telosnetwork/teloscan
# Remembering pagination state for tx table

# Fixes #293

## Description
When you find a special transaction from the table and you follow the link and then come back to the table, the pagination is lost and you must start from the first page again which leads to a terrible user experience.

This PR makes the table save in local storage the last pagination state and recovers it every time the table is used again.

## Test scenarios
To recreate this new behavior you must:
- go to the [home page](https://deploy-preview-336--teloscan-stage.netlify.app/)
- change the current page and the page size
- follow the link of any of the listed transaction
- press the Back button of your browser.
- the table should recover the last pagination state

## Checklist:
